topics/imaging/tutorials/tutorial-CP/tutorial.md

```diff
@@ -0,0 +1,633 @@
+>        - *"Handling of objects if excessive number of objects identified"*: `Continue`
+>
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > - The name entered to the input image and objects has to be consistent (case sensitive) with the names in *NamesAndTypes* and the tools to follow.
+>    > - The min and max diameter of the objects (`Typical minimum diameter of objects, in pixel units (Min)` and `Typical minimum diameter of objects, in pixel units (Max)`) will have to be adjusted to the resolution of the images.
+>    {: .comment}
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> We are using here Otsu's method for segmentation. What other segmentation options are available? What is the difference between them?
+>
+> > ### {% icon solution %} Solution
+> >
+> > In the *global* methods we have `Manual`, `Measurement`, `Minimum cross entropy`, `Otsu` and `Robust background`. For the *adaptive* ones we only have `Otsu`. Check the parameters' help to get more information on each one. 
+> >
+> {: .solution}
+>
+{: .question}
+
+From the previous tool, we got a group of objects (nuclei). Now, we want to export the segmentation masks as a single image to check how well the segmentation algorithm is performing. We also want to label the nuclei with their identifiers for future visual inspection of the results. The output of this step will look like:
+
+![Identified nuclei with labels](../../images/tutorial-CP/img_nuclei_labels.png "Identified nuclei with labels.")
+
+> ### {% icon question %} Questions
+>
+> Why are some nuclei not labeled in the image above?
+>
+> > ### {% icon solution %} Solution
+> >
+> > We have indicated in the tool **IdentifyPrimaryObjects** {% icon tool %} that the nuclei that are either outside the diameter range or touching the border should be discarded.
+> >
+> {: .solution}
+>
+{: .question}
+
+
+> ### {% icon hands_on %} Hands-on: Mask the nuclei detected
+>
+> 1. **ConvertObjectsToImage** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Select the input CellProfiler pipeline"*: `output_pipeline` (output of **IdentifyPrimaryObjects** {% icon tool %})
+>    - *"Enter the name of the input objects you want to convert to an image"*: `Nuclei`
+>    - *"Enter the name of the resulting image"*: `MaskNuclei`
+>    - *"Select the color format"*: `Binary (black & white)`
+>
+> 2. **DisplayDataOnImage** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Select the input CellProfiler pipeline"*: `output_pipeline` (output of **ConvertObjectsToImage** {% icon tool %})
+>    - *"Display object or image measurements?"*: `Object`
+>        - *"Enter the name of the input objects"*: `Nuclei`
+>        - *"Measurement category"*: `Number`
+>    - *"Enter the name of the image on which to display the measurements"*: `DNA`
+>    - *"Display mode"*: `Text`
+>        - *"Text color"*: `#ff0000`
```

Comment:
In the tool I add to pick a color (not to specify an hexa code). I am not sure everyone would know `#ff0000` is red.

Reply:
comment included on that :)